.eslintrc

```diff
@@ -24,21 +26,39 @@
       }
     ],
     "import/no-internal-modules": [
-      "warn", {
+      "warn",
+      {
         "allow": [
           "hyperview/**",
           "react-native/Libraries/StyleSheet/StyleSheetTypes"
         ]
       }
-    ]
+    ],
+    "max-len": [
+      "warn",
+      {
+        "code": 100
+      }
+    ],
+    "import/no-cycle": "off",
+    "lines-between-class-members": "off",
+    "no-else-return": "off",
+    "no-restricted-globals": "off",
+    "prefer-destructuring": "off",
+    "react/destructuring-assignment": "off",
+    "react/jsx-sort-props": "off",
```

Comment:
We follow a convention of sorting props alphabetically. This seems to bypass that if I'm not mistaken?

Reply:
Correct, though it is intended to be temporary. Upgrading to `eslint-plugin-instawork@0.4.0` resulted in 200+ new linting errors. I decided to leave fixing the errors to different PR - see #126 so this one could be kept as small as possible.